generics/services/users.js

```diff
@@ -9,7 +9,7 @@
 const request = require('request')
 const userServiceUrl = process.env.USER_SERVICE_URL
 
-const profile = function (token, userId = '') {
+const profile = function (userId = '') {
```

Comment:
@MallanagoudaB Hence you removed the token param from this function. make sure changes are made in each and every place in the service where the function is getting called